flower/api/control.py

```diff
@@ -62,9 +62,11 @@ def post(self, workername):
         if not self.is_worker(workername):
             raise web.HTTPError(404, "Unknown worker '%s'" % workername)
 
-        logger.info("Shutting down '%s' worker", workername)
-        self.capp.control.broadcast('shutdown', destination=[workername])
-        self.write(dict(message="Shutting down!"))
+        # logger.info("Shutting down '%s' worker", workername)
+        # self.capp.control.broadcast('shutdown', destination=[workername])
+        # self.write(dict(message="Shutting down!"))
```

Comment:
@NguyenHoangPhuTien Instead of commenting out the unused lines, it's better to remove them so it's not confusing when people read it and the code is not crowded with unused code.

Reply:
Removed Hans. Please help to re-check